x-pack/plugins/security/public/security_checkup/components/insecure_cluster_alert.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { MountPoint } from 'kibana/public';
+import { EuiText, EuiSpacer, EuiFlexGroup, EuiFlexItem, EuiButton } from '@elastic/eui';
+
+export const insecureClusterAlertTitle = i18n.translate(
+  'xpack.security.checkup.insecureClusterTitle',
+  { defaultMessage: 'Please secure your installation' }
+);
+
+export const insecureClusterAlertText = (onDismiss: () => void) =>
+  ((e) => {
+    render(
+      <div data-test-subj="insecureClusterAlertText">
+        <EuiText>
+          <FormattedMessage
+            id="xpack.security.checkup.insecureClusterMessage"
+            defaultMessage="Our free security features can protect against unauthorized access."
+          />
+        </EuiText>
+        <EuiSpacer />
+        <EuiFlexGroup>
+          <EuiFlexItem grow={false}>
+            <EuiButton
+              color="primary"
+              fill
+              href="https://www.elastic.co/what-is/elastic-stack-security"
+            >
+              Learn more
+            </EuiButton>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiButton onClick={onDismiss}>Dismiss</EuiButton>
```

Comment:
For the button name, I thought of "Skip", "No thanks", "Got it", and "Close".  Given all that, I prefer Dismiss.  Because we really don't want users to ignore this.  Or, can a toast have a Close icon in the upper right?

Reply:
> Given all that, I prefer Dismiss. we really don't want users to ignore this message.

Right, but I don't think we're adequately communicating to users that if they click Dismiss, they'll never see the message again.

> Or, can a toast have a Close icon in the upper right?

There is a close icon in the upper right of every toast -- currently if you click it for this toast, it dismisses the toast until the next page load. Unlike the button, which permanently hides the toast.

Reply:
For posterity: we settled on adding a "Don't show again" checkbox 😄 

![image](https://user-images.githubusercontent.com/5295965/94967873-ef07ba80-04cd-11eb-9b26-926b03f861ee.png)